### ISSUE TITLE
Recent docker build error

### ISSUE THREAD

**Issue by BorntraegerMarc** [libvips/libvips#1320]
Hello there 👋 

I know that this is probably the wrong place to file my bug report. But I'm a bit desperate since I can not pinpoint the origin of the libvips download error that I'm experiencing. So feel free to point me to the right direction and I'll be on my way 😄 

FYI: My builds have been working stable for more than a year. Only this week I've been seeing this error coming up. So I assume there were some external environment changes.

I'm building libvips for a docker linux alpine container like so:
```dockerfile
FROM node:8.12.0-alpine

RUN apk add vips vips-dev fftw-dev --update-cache --repository http://dl-3.alpinelinux.org/alpine/edge/testing/ --repository http://dl-3.alpinelinux.org/alpine/edge/main

RUN echo $'http://alpine.gliderlabs.com/alpine/edge/testing' >> /etc/apk/repositories && \
    apk update && \
    apk upgrade --update-cache --available && \
    apk add nodejs \
    xpdf \
    vips \
    vips-dev \
    python2 \
    git \
    make \
    g++
```

During our build process I receive the following error:
```java
Step 8/15 : RUN apk add vips vips-dev fftw-dev --update-cache --repository http://dl-3.alpinelinux.org/alpine/edge/testing/ --repository http://dl-3.alpinelinux.org/alpine/edge/main
 ---> Running in c21905072c39
fetch http://dl-3.alpinelinux.org/alpine/edge/main/x86_64/APKINDEX.tar.gz
fetch http://dl-3.alpinelinux.org/alpine/edge/testing/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.8/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.8/community/x86_64/APKINDEX.tar.gz
ERROR: unsatisfiable constraints:
  vips (missing):
    required by: world[vips]
  vips-dev (missing):
    required by: world[vips-dev]
ERROR: Service 'komed-image-processing' failed to build: The command '/bin/sh -c apk add vips vips-dev fftw-dev --update-cache --repository http://dl-3.alpinelinux.org/alpine/edge/testing/ --repository http://dl-3.alpinelinux.org/alpine/edge/main' returned a non-zero code: 2
Exited with code 1
```

Does anyone know why this error occurs?

**Comment by jcupitt** [comment]
Hello Marc, I think this is an alpine problem, their package dependencies have become a little mixed up during a transition.

I guess you are using sharp? I think the current sharp includes its own libvips binary, so you can just skip the vips install. Ask on the sharp tracker if you have problems.

**Comment by lovell** [comment]
It looks like the Alpine `vips-dev` package was recently promoted from `testing` to `community`.

https://github.com/alpinelinux/aports/commit/434ff132e2fa00b720f3dacb0b8079019496fe1d

https://pkgs.alpinelinux.org/packages?name=vips-dev&branch=edge

Happy to accept a PR to update the docs at https://github.com/lovell/sharp/blob/master/docs/install.md#alpine-linux

As John suggests, you could also let sharp install its own libvips binaries, which are made available for Alpine 3.8+.

**Comment by BorntraegerMarc** [comment]
Thanks @lovell and @jcupitt 

I updated the sharp docs to use the `community` edition:
https://github.com/lovell/sharp/pull/1730

Closing this issue...
